<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>E3t1</class>
 <widget class="QWidget" name="E3t1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="matWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="vecWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>80</y>
     <width>231</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="vecLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>531</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>解向量：</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>290</y>
     <width>391</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>方法选择</string>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>161</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>顺序Gauss消去法</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>列选主元Gauss消去法</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>全选主元Gauss消去法</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>181</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Gauss-Jordan消去法</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>matWidget</class>
   <extends>QWidget</extends>
   <header>matwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>vecWidget</class>
   <extends>QWidget</extends>
   <header>vecwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>vecLabel</class>
   <extends>QLabel</extends>
   <header>veclabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
